src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -226,16 +228,48 @@ public override async Task<dynamic> GetHostingEnvironmentPostBody(string hosting
             return hostingEnvironmentPostBody;
         }
 
+        public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceName, string query)
+        {
+            if (!query.StartsWith("\""))
+            {
+                // BUG: Passing as JSON requires query to be wrapped in quotes, fix API expected content type
+                query = $"\"{query}\"";
+            }
+
+            var request = new HttpRequestMessage(HttpMethod.Post, $"/api/service/{cloudServiceName}/invokesql?api-version=2.0")
+            {
+                Content = new StringContent(query, Encoding.Default, "application/json")
+            };
+
+            var response = await SendObserverRequestAsync(request);
+            response.EnsureSuccessStatusCode();
```

Comment:
In the case we get anything other than 200, lets also log the query that way we can backtrack to the logs to look for invalid SQL Queries.